yt_dlp/extractor/aflwomens.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+import re
+
+from .common import InfoExtractor
+
+from ..utils import urljoin
+
+
+class AFLWomensPlaylistIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?womens\.afl/(?P<id>match-(?:replays|highlights)|video/[a-z0-9-]+)$'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?womens\.afl/(?P<id>match-(?:replays|highlights)|video/[a-z0-9-]+)'
```

Reply:
Will the regex still be anchored to the end of the string? There are other paths under `/video` we _don't_ want to match here

Reply:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?womens\.afl/(?P<id>match-(?:replays|highlights)|video/[a-z0-9-]+)/?(?:[?#]|$)'
```
then do this to handle any fragments and query correctly